test/__init__.py

```diff
@@ -53,12 +54,22 @@ def run_role_meta_main(self, meta_main_text):
         return results
 
 
-def run_ansible_lint(cwd, role_path=None, bin=None, env=None):
-    """Run ansible-lint on a given path and returns its output."""
+def run_ansible_lint(
+        cwd: os.PathLike = None,
+        role_path: os.PathLike = None,
+        bin: str = None,
+        env: Dict = None):
```

Comment:
I'm pretty sure it's
```suggestion
        env: Dict[str, str] = None):
```